tests/e2e/release-e2e-findings.md

```
@@ -0,0 +1,161 @@
+...
+case "":
+    if hasToolCalls {
+        return "tool_use"
+    }
+```
+
+When a tool is forced via `tool_choice`, OpenAI (`gpt-4.1-nano`) returns
+`finish_reason: "stop"` **with** a non-empty `tool_calls` list — confirmed with a
+direct `/v1/chat/completions` call:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)